lib/mailtrap/client.rb

```
@@ -50,41 +44,93 @@ def send(mail)
       raise ArgumentError, 'should be Mailtrap::Mail::Base object' unless mail.is_a? Mail::Base
 
       request = post_request(request_url, mail.to_json)
-      response = http_client.request(request)
+      response = http_client(cache: true).request(request)
+      handle_response(response)
+    end
 
+    def send_batch(base, requests)
+      raise ArgumentError, 'hould be Mailtrap::Mail::Base object' unless base.is_a?(Mailtrap::Mail::Base)
+      raise ArgumentError, 'requests must be present' if !requests.is_a?(Array) || requests.empty?
```

RuboCop (1.75.5)

[convention] Add empty line after guard clause.

(Layout/EmptyLineAfterGuardClause)